osquery/core/windows/process_ops.cpp

```diff
@@ -70,4 +71,23 @@ boost::optional<std::string> getEnvVar(const std::string &name) {
 void cleanupDefunctProcesses() {}
 
 void setToBackgroundPriority() {}
+
+// Helper function to determine if thread is running with admin privilege.
+bool isUserAdmin() {
```

Comment:
This function checks if the current thread's token has elevated privs. Is this function name the most appropriate? Should it be something else? `runningAsAdmin`?